src/layouts/default.html

```diff
@@ -4,27 +4,20 @@
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
-  <head>
-    <link rel="stylesheet" type="text/css" href="{{root}}css/app.css">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width">
-    <title>{{subject}}</title>
-    <!-- <style> -->
-  </head>
-  <body>
-    <span class="preheader">{{description}}</span>
-    <table class="body">
-      <tr>
-        <td class="center" align="center" valign="top">
-          <center>
-            {{!-- Pages you create in the src/pages/ folder are inserted here when the flattened emails are created. --}}
-            {{> body}}
-          </center>
-        </td>
-      </tr>
-    </table>
-    <!-- prevent Gmail on iOS font size manipulation -->
-   <div style="display:none; white-space:nowrap; font:15px courier; line-height:0;"> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </div>
-  </body>
+<head>
+  <link rel="stylesheet" type="text/css" href="{{root}}css/app.css">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>{{subject}}</title>
+  <!-- <style> -->
+</head>
+<body>
+  <span class="preheader">{{description}}</span>
+  <wrapper class="body">
```

Comment:
I'm concerned that the `<center>` tag is missing here along with some of the classes for centering. Do you have any tests that show the container is centering works on all email clients supported?